source/_components/light.hue.markdown

```diff
@@ -27,6 +27,9 @@ If you want to enable the component without relying on the [discovery component]
 light:
   platform: hue
   host: DEVICE_IP_ADDRESS
+  allow_unreachable: true
+  allow_in_emulated_hue: false
+  allow_hue_groups: false
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.